api/bro_upload/utils.py

```diff
@@ -104,7 +104,7 @@ def create_delivery(
 
 def check_delivery_status(
     delivery_url: str, bro_username: str, bro_password: str
-) -> str:
+) -> requests.Response:
```

Comment:
Is een 'response' hier een handig antwoord? Ik zou gokken dat er in de functie zelf een `raise_for_status()` wordt gedaan en dat verder de delivery status terug wordt gegeven. Boolean? Of een constante?

Ik zou de response niet verder doorgeven tenzij het echt nodig is, want dan moet je op de plek waar je deze functie aanroept wel precies weten wat voor response je krijgt.